lib/process-services-cloud/src/lib/task/task-filters/components/edit-task-filters/base-edit-task-filter-cloud.component.ts

```diff
@@ -286,13 +288,13 @@ export abstract class BaseEditTaskFilterCloudComponent<T> implements OnInit, OnC
 
     onDateChanged(newDateValue: any, dateProperty: TaskFilterProperties) {
         if (newDateValue) {
-            const momentDate = moment(newDateValue, BaseEditTaskFilterCloudComponent.FORMAT_DATE, true);
+            const date = DateFnsUtils.formatDate(new Date(newDateValue), BaseEditTaskFilterCloudComponent.FORMAT_DATE);
```

Comment:
makes no sense, you first call new Date, then format to string, then convert to date again